docs/src/main/sphinx/release/release-368.md

```diff
@@ -0,0 +1,51 @@
+# Release 368 (xx Jan 2022)
+
+## General
+
+* Improve performance of arithmetic operations involving decimals with precision
+  larger than 18. ({issue}`10051`)
+* Introduce `prioritize-utilization` query execution policy that aims to improve
+  resource utilization for queries that involve joins and for concurrent
+  workloads. ({issue}`10350`)
+* Fix delegated refresh materialized views queries not transitioning into
```

Comment:
What's a "delegated refresh materialized view". That seems overly specific and not useful to readers. Also, instead of saying "not transitioning to RUNNING state", we should describe the impact. E.g., "Fix issue where `REFRESH MATERIALIZED VIEW` queries would get stuck" or some such.

Reply:
Reworded